export/index.template.html

```diff
@@ -742,7 +742,19 @@ <h2>Message log</h2>
           if (!TKHQ.verifyEnclaveSignature) {
             throw new Error("method not loaded");
           }
-          verified = await TKHQ.verifyEnclaveSignature(bundleObj.enclaveQuorumPublic, bundleObj.dataSignature, bundleObj.data.encappedPublic);
+
+          // Temporary solution to get the bytes signed by the enclave
+          var signedData = /"data":({[^}]+)/.exec(bundle);
+          if (signedData === null || signedData.length !== 2) {
+            throw new Error(`unable to find signed data in bundle: ${bundle}`);
+          }
+
+          // We add the closing brace back: it was our marker for the end of the group in the regex above, hence not captured.
+          const signedDataBytes = new TextEncoder().encode(signedData[1] + "}");
+          // Encode these bytes to hex since verifyEnclaveSignature expects hex-encoded bytes
+          const signedDataHexString = TKHQ.uint8arrayToHexString(signedDataBytes);
+
+          verified = await TKHQ.verifyEnclaveSignature(bundleObj.enclaveQuorumPublic, bundleObj.dataSignature, signedDataHexString);
```

Comment:
tiny nit: think it might make sense to rename the `publicKey` argument of `TKHQ.verifyEnclaveSignature` for consistency. but definitely a huge stamp of approval for now 🥳

Reply:
yeah that's fair. Let's rename for clarity!